src/SonarScanner.MSBuild.Tasks/IsTestByReference.cs

```diff
@@ -31,27 +32,31 @@ namespace SonarScanner.MSBuild.Tasks
     /// </summary>
     public sealed class IsTestByReference : Task
     {
-        // This list is duplicated in sonar-dotnet and sonar-security and should be manually synchronized after each change.
-        internal /* for testing */ static readonly ISet<string> TestAssemblyNames = new HashSet<string>
+        /* This list is duplicated in sonar-dotnet and sonar-security and should be manually synchronized after each change.
+        /
```

Comment:
the comment style is a bit weird (non-standard),  I would expect either to see `* line ` or `line`... for a multi-line comment

this is what my IDE generates:
```
        /**
         *
         */
```